test/verify/check-image

```diff
@@ -560,9 +560,10 @@ class TestLargeImage(composerlib.ComposerCase):
                         "Image build complete")
         # log should contains rpm, hostname, users stages and tar assembler
         b.click("li[aria-labelledby={}] button:contains('Logs')".format(selector))
-        b.wait_in_text("#{}-logs".format(uuid), "Stage org.osbuild.rpm")
-        b.wait_in_text("#{}-logs".format(uuid), "Stage: org.osbuild.rpm-ostree")
-        b.wait_in_text("#{}-logs".format(uuid), "Assembler org.osbuild.ostree.commit")
+        # temporarily disable log contents checking
+        # b.wait_in_text("#{}-logs".format(uuid), "Stage org.osbuild.rpm")
+        # b.wait_in_text("#{}-logs".format(uuid), "Stage: org.osbuild.rpm-ostree")
+        # b.wait_in_text("#{}-logs".format(uuid), "Assembler org.osbuild.ostree.commit")
```

Comment:
I think cockpit-composer should not assert on the contents of the logs (but composer should!).

Reply:
So you think these log checks should be permanently removed?

Reply:
Yeah, I just spoke about that with Tom, and we suggest removing these checks.

Reply:
Yes :) The logs are meant to be human readable only, so we should treat them as opaque and not assert on their contents (could verify that they are not empty though!). Reason being that they may change at any time in the future.

This did uncover a real bug though! So this should be fixed in composer, and tests should be added there to verify the logs are as expected (even if opaque they should be useful...). Then if ever composer changes the format of the logs, only the composer tests would have to be adjusted.

Reply:
Okay will fully remove these.